### PR TITLE
feat: Added Question and Result Pages

### PR DESCRIPTION
**Pull request by jalajcodes**
Demo:
https://deploy-preview-4--max-quiz.netlify.app/pages/question.html

https://deploy-preview-4--max-quiz.netlify.app/pages/result.html

### CODE REVIEW

**Review by ArjunJunna — APPROVED** [review]
code is clean. Good work on that.

File: pages/question.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+          <h2>Quiz Name</h2>
+        </div>
+        <div class="quiz__info">
+          <span>Question <em>5/5</em></span>
+          <span>Score <em>100</em></span>
+        </div>
+        <div class="quiz__question">
+          <h4>Where was this movie shot?</h4>
+        </div>
+        <div class="quiz__options">
```
Comment (nayyyhaa) [inline pages/question.html]: extra divs can be removed & styling can be applied to the below semantic tags.

File: styles/question.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.question {
+  display: grid;
+  place-items: center;
+}
+
+.quiz {
+  display: flex;
```
Comment (nayyyhaa) [inline styles/question.css:7]: you can create one separate flex class & use it for reusability.

**Review by nayyyhaa — APPROVED** [review]
I've mentioned a few points, look into that. Nice work tho, your UI looks clean & amazing. Keep it up! 💯

**Pull request merged**